src/cftlib/java/uk/gov/hmcts/divorce/cftlib/RunWithCFT.java

```diff
@@ -1,19 +0,0 @@
-package uk.gov.hmcts.divorce.cftlib;
-
-import uk.gov.hmcts.divorce.CaseApiApplication;
-import uk.gov.hmcts.rse.ccd.lib.api.LibRunner;
-
-import java.util.Map;
-import java.util.Set;
-
-@SuppressWarnings("HideUtilityClassConstructor") // Spring needs a constructor, its not a utility class
-public class RunWithCFT {
-
-    public static void main(final String[] args) {
-        LibRunner.run(CaseApiApplication.class, Set.of(), Map.of(
```

Comment:
I didn't mind having this class as it made it a bit more obvious how we could tweak and override bits for specific CCD apps

Reply:
The LibRunner is gone because we have a new bootstrapping process. These were always global overrides rather than service-specific however. Can add service-specific overrides in future.